### ISSUE TITLE
Content support for Silent Failures work

### ISSUE THREAD

**Issue by strelichl** [department-of-veterans-affairs/va.gov-team#95507]
## Issue description

This ticket captures all the work being done to address silent failures issues. Use the ticket comments to note any tasks completed or decisions made during the current sprint.

---
## Acceptance Criteria
- [x] Work tracked in comments
- [x] New ticket opened for next sprint

**Comment by laurwill** [comment]
Yesterday 10/21 I spent my entire work day and I think Megan also spent her entire work day on these tasks:
- Syncing with VA Notify and Jeana from VFF to align templates in GitHub for 5 silent-failure-related emails, and doing pull requests to fix template content and add guidance to help teams fill in variables in those emails. The templates had gotten out of whack between GitHub, Sharepoint, and VA Notify system versions, and we had to resolve some formatting and language questions.
- Creating a list of plain language form names for teams to use in these emails. This work is ongoing, hope to wrap that today.

Also noting that previous to yesterday:
- Laura worked with VFF, Aliyah, Danielle, VA Notify to get the original Sharepoint versions of these emails ready
- Laura and Megan did a pull request to update the new pattern page about form submission status related to these emails

@strelichl this list is a little informal and involves several team members, but in general I would record this as a giant extra-large content task!

**Comment by megzehn** [comment]
Update: @laurwill and I worked with VA Notify and Veteran Facing Forms teams to come up with documentation for product teams working on emails for ZSF initiative. 

Documenting those resources here:

-[General content guidance for email and text notifications](https://design.va.gov/content-style-guide/email-and-text-notifications)
-[Pattern guidance to help users to stay informed of their form status](https://design.va.gov/patterns/help-users-to/stay-informed-of-their-application-status)  
-[Form status email templates](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/tree/main/Design/patterns/Application%20status/VA-notify-email) (Github link for visibility, but we link to the VA Notify templates in VADS)
-[Table with plain language form names](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/blob/main/Design/patterns/Application%20status/VA-notify-email/all-form-names.md) 

Product teams are actively using these templates. We've also worked with a smaller subset of product teams to customize these templates. 

@aprocik1 we can move this to validate and close it out!
